src/components/common/Extensions/UserHub/partials/StakesTab/partials/StakesList.tsx

```diff
@@ -29,7 +30,7 @@ const StakesList = ({ stakes, loading, colony }: StakesListProps) => {
   }
 
   return (
-    <>
+    <div className={styles.stakesListContainer}>
```

Comment:
I would suggest to apply all classes that are actually supported by tailwind.

Reply:
would something like:
` <div className={clsx(styles.stakesListContainer, 'pr-2 .....')}>` suffice? I would love to rework the scrollbar while I'm at it, but this PR is a bit old, so I'm getting merge conflicts every time anybody touches any file tracking was added to :sweat_smile: 
Can we open an issue to tackle that, I can do it whenever, since the package you've linked looks like it should do the job